src/http.rs

```diff
@@ -1,9 +1,12 @@
 // Copyright 2018 the Deno authors. All rights reserved. MIT license.
 
-use errors::DenoResult;
+use errors;
+use errors::{DenoError, DenoResult};
+use futures;
+use futures::future::Either;
+use futures::Future;
 use tokio_util;
 
-use futures::Future;
 use futures::Stream;
 use hyper;
 use hyper::client::Client;
```

Comment:
Imports are broken into two sections, the first one are local imports - eg `use errors::{DenoError, DenoResult};`. The second one are third party crates. So please reorder appropriately.

Reply:
Oh okay